### PR TITLE
Added a comment

### PR DESCRIPTION
**Pull request by SpaceAndAlsoTime**

### CODE REVIEW

**Comment by peopleoff** [comment]
This comment did not follow our style guide, please re-fix

File: readit.js [hunk 1]
```diff
@@ -50,3 +50,4 @@ client.on('message', msg => {
 client.login(auth.token);
 
 //Implemented new feature
+//Added comment
```
Comment (peopleoff) [inline readit.js]: Bad Comment

Comment (SpaceAndAlsoTime) [reply readit.js]: I fixed it

**Comment by SpaceAndAlsoTime** [comment]
I'm done

**Comment by peopleoff** [comment]
Looks Neato

**Pull request merged**